data/json/items/tool/lighting.json

```diff
@@ -528,8 +528,8 @@
     "material": [ "clay" ],
     "symbol": ";",
     "color": "brown",
-    "ammo": [ "lamp_oil" ],
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "watertight": true, "ammo_restriction": { "lamp_oil": 250 } } ],
+    "ammo": [ "crude_lamp_oil" ],
```

Comment:
Could it use both kerosene and crude oil?

Reply:
Sure thing, like this?
`"ammo": [ "crude_lamp_oil", "lamp_oil" ],
    "pocket_data": [ { "pocket_type": "MAGAZINE", "watertight": true, "ammo_restriction": { "crude_lamp_oil": 250, "lamp_oil": 250 } } ],`

Reply:
I think yes, look at https://github.com/CleverRaven/Cataclysm-DDA/blob/0c6d3025a572972e41b3891a93c9e9bbdeeedea9/data/json/items/gun/10mm.json#L29 for an exemple

Reply:
yeah thats exactly the one i looked at as well, actually xD

Reply:
just check if it loads in game and that you can use the lamp with both ammo type and we should be good

Reply:
tested, both worked